app/domain/util/open_ssl/x509/smart_csr.rb

```diff
@@ -29,6 +29,15 @@ def common_name
           smart_subject.common_name
         end
 
+        def common_name=(common_name)
+          smart_subject.common_name = common_name
+        end
+
+        # We don't just return subject as it might have been updated
+        def subject_to_s
```

Comment:
This method name doesn't read very intuitively.  Can we just call it `subject`?

Reply:
well, it can't be `subject` because that is another parameter - the one that smart_subject wraps (https://github.com/cyberark/conjur/blob/528c2eac8096807a19245ad3bbed6c7e472014f2/app/domain/util/open_ssl/x509/smart_csr.rb#L35)

I thought of `updated_subject` but that can be misleading because it might not have been updated.

`printable_subject` is also a candidate but that's very similar to `subject_to_s`. 

what do you think?